cypher/cypher-docs/src/docs/dev/ql/administration/databases-command-syntax.asciidoc

```diff
@@ -17,14 +17,21 @@ YIELD { * \| field[, ...] } [ORDER BY field[, ...]] [SKIP n] [LIMIT n]
 | Create Database
 | [source]
 CREATE DATABASE name [IF NOT EXISTS]
+[TOPOLOGY n PRIMAR{Y\|IES} [m SECONDAR{Y\|IES}]]
 [OPTIONS "{" option: value[, ...] "}"]
 [WAIT [n [SEC[OND[S]]]]\|NOWAIT]
 
 [source]
-CREATE OR REPLACE DATABASE name [WAIT [n [SEC[OND[S]]]]\|NOWAIT]
+CREATE OR REPLACE DATABASE name
+[TOPOLOGY n PRIMAR{Y\|IES} [m SECONDAR{Y\|IES}]]
+[WAIT [n [SEC[OND[S]]]]\|NOWAIT]
 |Alter Database
 | [source]
-ALTER DATABASE name [IF EXISTS] SET ACCESS {READ ONLY \| READ WRITE}
+ALTER DATABASE name [IF EXISTS]
+{
+SET ACCESS {READ ONLY \| READ WRITE} \|
+SET TOPOLOGY n PRIMAR{Y\|IES} [m SECONDAR{Y\|IES}]
```

Comment:
did we add a separate sub-privilege for this?

cause we have since before both an `ALTER` and `SET ACCESS` privilege... and should therefore probably have added a `SET TOPOLOGY` on the same level as `SET ACCESS`... (and then it should also be documented)

Reply:
as we did not add a separate privilege in the implementation it doesn't need to be documented